variables.tf

```diff
@@ -1,34 +1,11 @@
-variable "addon_context" {
```

Comment:
https://github.com/terraform-linters/tflint is great for finding un-used variables/locals and other common conventions

Reply:
I dug into this and am grateful for you spotting this and reporting it!

The Superlinter [config](https://github.com/hashicorp/terraform-aws-hashicorp-vault-eks-addon/blob/main/.github/workflows/superlinter.yml#L27) for this repository has `VALIDATE_TERRAFORM` enabled and that used to take care of `tflint`. 

As it turns out, in `v4` of Superlinter, `VALIDATE_TERRAFORM_TFLINT` is the correct way for `tflint` to be enabled, as more granular control was added.

I've fixed this in the long-running PR of https://github.com/hashicorp/terraform-aws-hashicorp-vault-eks-addon/pull/3 and also merged this after brief discussion.

For other readers: #3 has a couple of `tflint` warnings that are fixed by this PR (#8)